articles/client-auth/v2/mobile-desktop.md

```diff
@@ -0,0 +1,394 @@
+---
+title: Authentication for Mobile & Desktop Apps
+description: Explains how to authenticate users in a mobile or desktop application.
+toc: true
+---
+
+# Authentication for Mobile & Desktop Apps
+
+You can authenticate users of your mobile/desktop applications by:
+
+* Using the [Lock](/libraries/lock) client libraries;
+* Calling the Auth0 [Authentication API](/api/authentication) endpoints.
```

Comment:
By the way, isn't auth0.js a valid option? Maybe follow up with Jeff or crew-brucke.